<launch>

<!-- Establish communication between Arduino and ROS -->
<node name="serial_node" pkg="rosserial_arduino" type ="serial_node.py" output="screen" >
  <param name="port" value="/dev/ttyACM0" />
</node>

<!-- main program -->
<node name="msd_arm_main" pkg="msd_arm" type ="main.py" output="screen"/>

<!-- convert angle of robotic arm's joints from radian to degree -->
<node name="convertAngle" pkg="msd_arm" type ="convertAngle.py"/>

<!-- launch moveit service -->
<include file="$(find urdf_arm_config)/launch/demo.launch" />

</launch>
